python/pyiceberg/schema.py

```diff
@@ -169,6 +169,9 @@ def find_type(self, name_or_id: Union[str, int], case_sensitive: bool = True) ->
             raise ValueError(f"Could not find field with name or id {name_or_id}, case_sensitive={case_sensitive}")
         return field.field_type
 
+    def find_last_field_id(self):
```

Comment:
In Java, this is `highestFieldId`. Can we do the same here?

```
@property
def highest_field_id(self):
    ...
```

Reply:
Sure thing, updated 👍🏻